crates/hive-utils/src/madara/utils.rs

```diff
@@ -257,37 +266,35 @@ mod tests {
 
         // Create an atomic reference to the test environment to avoid dropping it
         let env = Arc::clone(&test_environment);
-        // It is not possible to block the async test task, so we need to spawn a blocking task
-        tokio::task::spawn_blocking(move || {
```

Comment:
Mainly removed `spawn_blocking` and manually dropping the lock after we set all values in storage